dotcom-rendering/src/components/marketing/banners/designableBanner/DesignableBannerV2.tsx

```diff
@@ -450,7 +450,7 @@ const DesignableBannerV2: ReactComponent<BannerRenderProps> = ({
 								setThreeTierChoiceCardSelectedProduct
 							}
 							choices={getChoiceCardData(
-								isTabletOrAbove,
+								false,
```

Comment:
This field is `longerBenefits: boolean`
so `false` means always show the shorter benefits copy